blockball-bukkit-plugin/src/main/java/com/github/shynixn/blockball/bukkit/logic/business/listener/BallListener.kt

```diff
@@ -65,7 +64,7 @@ class BallListener @Inject constructor(
      */
     @EventHandler
     fun onChunkSaveEvent(event: ChunkUnloadEvent) {
-        event.chunk.entities.filter { e -> e is ArmorStand }
+        event.chunk.entities.filter { e -> e.customName == "ResourceBallsPlugin" }
```

Comment:
CustomName is a dangerous method which can return null. This has to get solved differently.

Reply:
Does the "e.customName == "ResourceBallsPlugin" return false or throw a NullPointException in Kotlin? I am actually not sure right now.

Reply:
AFAIK equals check in Kotlin is null-safe.
https://kotlinlang.org/docs/reference/operator-overloading.html#equals